src/commands/api-mesh/create.js

```diff
@@ -150,23 +150,28 @@ class CreateCommand extends Command {
 								adobeIdIntegrationsForWorkspace.apiKey,
 							);
 
-							const { meshURL } = await getMesh(imsOrgId, projectId, workspaceId, mesh.meshId);
+							const { meshURL } = await getMesh(
+								imsOrgId,
+								projectId,
+								workspaceId,
+								workspaceName,
+								mesh.meshId,
+							);
 							const meshUrl =
 								meshURL === '' || meshURL === undefined
 									? MULTITENANT_GRAPHQL_SERVER_BASE_URL
 									: meshURL;
 
 							if (
-								meshUrl === TMOConstants.TMO_STAGE_URL ||
-								meshUrl === TMOConstants.TMO_SANDBOX_URL ||
-								meshUrl === TMOConstants.TMO_PROD_URL
+								adobeIdIntegrationsForWorkspace.apiKey &&
+								MULTITENANT_GRAPHQL_SERVER_BASE_URL.includes(meshUrl)
```

Comment:
Please pull in the latest changes that went into `develop` since it contains changes to the `adobeIdIntegrationsForWorkspace.apiKey`